pythran/tests/test_distutils.py

```diff
@@ -14,37 +16,66 @@
 else:
     so_version = ""
 
+def merge(x, y):
+    z = x.copy()
+    z.update(y)
+    return z
+
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
+    except OSError as exc:  # Python >2.5
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+
```

Comment:
please add a comment that explain why you're doing that. Probably take it from the commit message